indico/modules/rb/templates/booking_list.html

```diff
@@ -82,7 +88,7 @@
         {% endif %}
         {% if links %}
             <div>
-                <h3>Room bookings</h3>
+                <h3>Room booking occurrences</h3>
```

Comment:
Not sure if that's necessary. Yes, technically it's occurrences. But for the end user an occurrence is a time where they have*booked a room*....